miner/worker.go

```diff
@@ -652,6 +652,9 @@ func (env *Work) commitTransactions(mux *event.TypeMux, transactions types.Trans
 	if len(coalescedLogs) > 0 {
```

Comment:
Although then I'd do it like this:

```go
	if len(coalescedLogs) > 0 || env.tcount > 0 {
		go func() {
			if len(coalescedLogs) > 0 {
				mux.Post(core.PendingLogsEvent{Logs: coalescedLogs})
			}
			if env.tcount > 0 {
				mux.Post(core.PendingStateEvent{})
			}
		}()
	}
```

I.e. don't spin up the goroutine if there isn't anything to do.